src/lib/php/Dao/LicenseDao.php

```diff
@@ -713,4 +713,34 @@ public function getLicenseObligations($licenseLists, $candidate = false)
       return $ObligationRef;
     }
   }
+
+  /**
+   * @param string $licenseShortname
+   * @param int|null $groupId
+   * @return null
+   */
+  public function deleteLicenseByShortName($licenseShortname, $groupId=null)
+  {
+    $extraCondition = "";
+    $condition = 'rf_shortname=$1';
+    $param = array($licenseShortname);
+    $this->dbManager->deleteSingleRow(
+      "DELETE FROM license_ref WHERE $condition",
+      $param, __METHOD__ . ".$condition.only");
```

Comment:
This deletion should be conditional? Since `license_candidate` inherits `license_ref`, doing a delete here will make following condition redundant.